plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -1,11 +1,31 @@
 import { makeExecutableSchema, IResolvers } from "graphql-tools";
-import { Observable } from "rxjs";
+import { Observable } from "rxjs/Observable";
```

Comment:
With this import of observable, you need to also import all the other pieces you use, like throw, map timer.

https://www.learnrxjs.io/concepts/operator-imports.html

something like:

import  "rxjs/add/operator/map"

The only reason this code now works is that some other dependency of it does `import { Observable } from "rxjs";` and imports everything.